### PR TITLE
Added Nusphere DevNet as a testnet.

### PR DESCRIPTION
**Pull request by aniketpatil8824**
Nusphere DevNet added for Kaleido blockchain.

Please review our newly added Nusphere DevNet and add it to the Snapshot networks.

### CODE REVIEW

**Comment by aniketpatil8824** [comment]
Hi @ChaituVR, do i need to add the network details manually into https://chainlist.org/?testnets=true&search=1733320111 link? i mean in the Add Your Network & Add Your RPC subsections in the following links
1. https://github.com/ethereum-lists/chains/tree/master/_data/chains
2. https://github.com/DefiLlama/chainlist/blob/main/constants/extraRpcs.js

There is an official Kaleido documentation available for your reference as below.
https://docs.kaleido.io/

Let me know about the same.

Thanks & Regards,
Aniket.

File: src/networks.json [hunk 1]
```diff
@@ -1756,6 +1756,26 @@
     "start": 7521509,
     "logo": "ipfs://QmNnGPr1CNvj12SSGzKARtUHv9FyEfE5nES73U4vBWQSJL"
   },
+  "1733320111": {
+    "key": "1733320111",
+    "name": "Nusphere DevNet",
+    "chainId": 1733320111,
+    "network": "Nusphere DevNet",
+    "testnet": true,
+    "multicall": "0x7b55399d0b394b32b6f1974892be3f8d9e659475",
```
Comment (ChaituVR) [inline src/networks.json:1765]: i can't find multicall contract call code here? https://nusphere-testnet.tryethernal.com/address/0x7b55399d0b394b32b6f1974892be3f8d9e659475?tab=transactions

Comment (aniketpatil8824) [reply src/networks.json:1765]: Hi @ChaituVR, i can see the contract call code. https://nusphere-testnet.tryethernal.com/transaction/0xb07b11882b17dba37e98fb8f8f7292e569359bf452dc48eaef88a8fed05fb932 as well as https://nusphere-testnet.tryethernal.com/address/0x7b55399d0b394b32b6f1974892be3f8d9e659475?tab=transactions also.

Comment (aniketpatil8824) [reply src/networks.json:1765]: contract_call_code

contract_nft

File: src/networks.json [hunk 2]
```diff
@@ -1756,6 +1756,26 @@
...
+    "rpc": [
+      {
+        "url": "https://u0p4xgo08c-u0mrdoctj2-rpc.us0-aws.kaleido.io/",
+        "user": "u0bzotd8yx",
+        "password": "610jGOOaAMlgzFRic40_QueqbJolGulwo0r9JiuE9VQ"
+      }
+    ],
+    "explorer": {
+      "url": "https://nusphere-testnet.tryethernal.com/overview"
```
[the remark this answers is not in the document]
Comment (aniketpatil8824) [reply src/networks.json]: okay. i will edit the url as per your suggestion.

File: src/networks.json (hunk 1 above)
Comment (aniketpatil8824) [reply src/networks.json:1765]: Hi. I have made some changes in the following repos. should i make a pull request of the following repos?

1. https://github.com/ethereum-lists/chains/compare/master...aniketpatil8824:chains:master
2. https://github.com/DefiLlama/chainlist/compare/main...aniketpatil8824:chainlist:main

Comment (aniketpatil8824) [reply src/networks.json:1765]: Hi @ChaituVR, one thing I observed is, in the Ethernal Explorer the network name is 'Nusphere Testnet' as you can see in the above screenshots. In the PR, i have given name of the network as 'Nusphere DevNet'. Is it the reason that you are unable to see the network and ultimately the contract in the network? Let me know.

Comment (aniketpatil8824) [reply src/networks.json:1765]: https://nusphere-testnet.tryethernal.com/contract/0x7b55399d0b394b32b6f1974892be3f8d9e659475?tab=code 

This multicall contract has been verified on explorer. Using above URL, under the code section, we can verify the contract. 

A screenshot of contract verification is as given below.

verify_PLNCTZ

Let me know if you are also able to view it or not. Thanks.

Comment (aniketpatil8824) [reply src/networks.json:1765]: We have to flatten the NFT contract as it contains Openzeppelin library imports.  
Below three screenshots describes how to verify the contract. 

verify-1

verify-2

NFT-ABI

In the Constructor Arguments we have to pass the ABI of the deployed PlanetXCitizen contract as shown in the screenshot.

Comment (ChaituVR) [reply src/networks.json:1765]: Is there some way to check the contract code on your explorer?

Comment (aniketpatil8824) [reply src/networks.json:1765]: As of now on the explorer I can't see any way to check the contract code. 
We can see the contract creation data after contract creation using below link.
https://nusphere-testnet.tryethernal.com/transaction/0xb07b11882b17dba37e98fb8f8f7292e569359bf452dc48eaef88a8fed05fb932

Comment (aniketpatil8824) [reply src/networks.json:1765]: Hi. Should we share the NFT multicall contract OR GitHub repo with you? Right now there is no way to check the contract code on explorer. Do let me know.

Comment (ChaituVR) [reply src/networks.json:1797]: Hi @aniketpatil8824 There should be a way to check multicall code on explorer, else we won't be able to add this network

**Comment by ChaituVR** [comment]
Closing this as stale, feel free to reopen if needed

**Pull request closed without merging**